logging/logging.go

```diff
@@ -254,6 +254,7 @@ type Logger struct {
 	populateSourceLocation int
 	partialSuccess         bool
 	redirectOutputWriter   io.Writer
+	minimalLoggedSeverity  Severity
```

Comment:
Is this a breaking change? Could this have any affects on existing users?

Reply:
No. Severity zero value is zero = LogSeverity_DEFAULT. That's why [this if](https://github.com/googleapis/google-cloud-go/pull/6346/files#diff-6bb8ce03ad8192446d441a3d85bf0fb8307e3d7829bc027785b35ce95868da70R664) will be skipped if minimalLoggedSeverity is not set. It's backward compatible change.